source/_components/weblink.markdown

```diff
@@ -2,14 +2,14 @@
 layout: page
 title: "Weblink"
 description: "Instructions how to setup Links within Home Assistant."
-date: 2016-02-02 20:00
+date: 2018-01-19 20:00
 sidebar: true
 comments: false
 sharing: true
 footer: true
 logo: home-assistant.png
 ha_category: Front end
-ha_release: 0.13
```

Comment:
Do not update the HA release on module update. It indicates the initial HA release, not the last updated one.